_drafts/our_history.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "History"
+description: "Why did we founded Triple D?"
+header-img: "img/laptop-bg.png"
+---
```

Comment:
Is het de bedoeling dat deze pagina ook al online komt ?
zo niet dan moet je die headers verwijderen, anders maakt die volgens mij gewoon een link op de homepage